mne/viz/topo.py

```diff
@@ -140,9 +153,10 @@ def format_coord_multiaxis(x, y, ch_name=None):
             ax = plt.axes(pos[idx])
             ax.patch.set_facecolor(axis_facecolor)
             plt.setp(list(ax.spines.values()), color=axis_spinecolor)
-            ax.set(xticklabels=[], yticklabels=[])
-            plt.setp(ax.get_xticklines(), visible=False)
-            plt.setp(ax.get_yticklines(), visible=False)
+            if not do_legend:
+                ax.set(xticklabels=[], yticklabels=[])
+                plt.setp(ax.get_xticklines(), visible=False)
+                plt.setp(ax.get_yticklines(), visible=False)
```

Comment:
The legend only makes sense if you have ticks on the axes really.

Reply:
same comment as above about state machine vs obj-oriented approach:
```py
tklines = list(ax.get_xticklines()) + list(ax.get_yticklines())
list(tk.set_visible(False) for tk in tklines)
```